src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,55 @@
+.calendar {
+  display: flex;
+  position: absolute;
+  flex-direction: row;
+  flex-wrap: wrap;
+  gap: 1px;
+  width: 710px;
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%, -50%);
+
+  &__block {
+    box-sizing: border-box;
+    display: flex;
+    height: $cell-size;
+    width: $cell-size;
+    border: 1px solid black;
+    background-color: #eee;
+    font-size: 30px;
+    color: black;
+    justify-content: center;
+    align-items: center;
+    cursor: pointer;
+  
+  
+    &:hover {
+      display: flex;
+      background-color: pink;
+      transform: translateY(-20px);
+      transition: transform 0.5s ;
```

Comment:
move transition to `&__block`
<img width="944" alt="image" src="https://user-images.githubusercontent.com/102411130/223350272-85e63378-a3d9-4407-9cd5-0302e09f868d.png">